operator/v1/0000_70_cluster-network-operator_01.crd.yaml

```diff
@@ -290,9 +290,31 @@ spec:
                         type: integer
                     type: object
                   ovnKubernetesConfig:
-                    description: oVNKubernetesConfig configures the ovn-kubernetes
-                      plugin. This is currently not implemented.
+                    description: ovnKubernetesConfig configures the ovn-kubernetes
+                      plugin.
                     properties:
+                      gatewayConfig:
+                        default:
+                          mode: shared
+                        description: gatewayConfig holds the configuration for node
+                          gateway options. If unset, reported defaults are used. Supported
+                          from OCP 4.10.
+                        properties:
+                          disableSNATMultipleGWs:
+                            default: false
+                            description: disableSNATMultipleGws sets whether to disable
+                              SNAT of egress traffic in namespaces annotated with
+                              routing-external-gws. Default is false.
+                            type: boolean
+                          mode:
```

Comment:
s/mode/gatewayMode

Reply:
looks like it's under gatewayConfig, so we may be OK with just calling the property `mode`

Reply:
ah, yeah, I had suggested `gatewayMode` if we dropped the `gatewayConfig` struct, but if we still have that struct then just `mode` is fine